config/prod/nf-syn23664726-s3.yaml

```diff
@@ -21,6 +21,8 @@ parameters:
     - 'arn:aws:sts::526515999252:assumed-role/AWSReservedSSO_S3ExternalCollab_40c062f682e7f3f5/robert.allaway@sagebase.org'
     - 'arn:aws:iam::694704239947:root' ## Nebula Genomics access
     - 'arn:aws:iam::694704239947:user/giri' ## Nebula Genomics access
```

Comment:
should we remove one of these from the list?

Reply:
```suggestion
    - 'arn:aws:iam::694704239947:root' ## Nebula Genomics access
```

Reply:
Yes! Done.